easybuild/easyconfigs/a/ADIOS/ADIOS-20210804-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,73 @@
+# Based on ADIOS-1.13.1-foss-2019a-Python-2.7.15.eb
+# Updated to foss-2020a toolchain and switched to ConfigureMake
+# Latest version from GitHub
+# Mini-XML updated to latest 2.x. version
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'ADIOS'
+version = '20210804'
```

Comment:
I don't really understand which version this is, also because the comment at the top says "Latest version from GitHub". They seem to offer much more recent versions on their GitHub page than what this line is suggesting. Could you elaborate?

Reply:
The most recent commit to the GitHub repository [1] was made on 04 Aug 2021.
`ADIOS2` has of course more recent development, but it is not back-compatible and `GKeyll` requires `1.x`

[1] https://github.com/ornladios/ADIOS

Reply:
Ah, sorry, I was looking at the wrong GitHub page (ADIOS2 instead of ADIOS) yesterday :man_facepalming: